internal/website/content/howto/pubsub/publish.md

```diff
@@ -106,7 +106,7 @@ create an [AWS session][] with the same region as your topic:
 [`awssnssqs.OpenSQSTopic`]: https://godoc.org/gocloud.dev/pubsub/awssnssqs#OpenSQSTopic
 [AWS session]: https://docs.aws.amazon.com/sdk-for-go/api/aws/session/
 
-## Google Cloud Pub/Sub {#gcp}
+### GCP Cloud Pub/Sub {#gcp}
```

Comment:
Isn't this called Google Cloud Pub/Sub? Why the change?

Reply:
Changing back